### ISSUE TITLE
TIF files sometimes break image uploader

### ISSUE THREAD

**Issue by glennlunder** [demozoo/demozoo#209]
In some cases, with certain TIF files (but not all), the image uploader never completes, but stays in processing mode forever with the spinning wheel. Discussed this with Gasman on Slack yesterday, and he indicated we may be a few versions behind on our Image processing library (Pillow), and that it may be a bug in that which may be fixed by now.

I see there was TIF work in Pillow 2.7.0 and 3.0.0 at least, but the release note from the homepage does not list release dates so I do not know the age of those updates. The latest version of Pillow is 3.0.0; http://python-pillow.org/

Example files for testing;

https://files.scene.org/view/parties/1999/gravity99/grtc/bfield.rar
https://files.scene.org/view/parties/2002/euskal10/grtc/07primeraviso.zip

**Comment by DannyCork** [comment]
I tested against staging, a regular image uploaded successfully , The tif in Bfield.rar failed to upload, the production page returned however the screenshot was not returned.

Pillow is now at 5.4.1
https://github.com/python-pillow/Pillow/blob/f38f01bbe3a0a9f49ce592c86ff20c01c9655133/docs/releasenotes/5.4.1.rst

Locally, I removed Pillow, changed requirements.txt to allow the latest and installed the Pillow 5.4.1. I was however unable to upload any images as my installation was complaining while attempting to upload. 

To summarise, I was unable to test Pillow 5.4.1 :(
